packages/nxengine/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_NAME=NXEngine
+TERMUX_PKG_HOMEPAGE=https://nxengine.sourceforge.net
+TERMUX_PKG_DESCRIPTION="Open-source rewrite engine of the Cave Story for Dingux and MotoMAGX"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@Yonle"
+TERMUX_PKG_VERSION=1.0.0.4-Rev4
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_DEPENDS="sdl, sdl-ttf, pulseaudio"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_get_source() {
+	git clone https://github.com/EXL/NXEngine $TERMUX_PKG_SRCDIR
+}
+
+termux_step_make() {
+	export TERMUX_PREFIX
```

Comment:
It has not been exported, right? Remove it.

Reply:
Alright